windows_docker_resources/Dockerfile.dashing

```diff
@@ -58,7 +58,9 @@ RUN C:\TEMP\python-37.exe /quiet `
 RUN powershell -noexit "Set-ExecutionPolicy Bypass -Scope Process -Force; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))"
 
 # choco installs
-RUN choco install -y cmake curl git vcredist2013 vcredist140 cppcheck patch
+RUN choco install -y cmake curl git vcredist2013 vcredist140 patch
+# Pin cppcheck until our builds are compatible with 2.0 https://github.com/ros2/ros2/issues/942
+RUN choco install -y cppcheck --version 1.90
```

Comment:
Line 112 also needs this. Similar to the rigamarole we had to go through for 1.89->1.9. I'll commit this after dinner if you're not around.

https://github.com/ros2/ci/blob/365f235717b912fceb60768751ba1856434d1aaf/windows_docker_resources/Dockerfile.msvc2019#L103

Reply:
Would it make more sense to pin cppcheck. I'll propose something.

Reply:
What do you think of 26da4cbf6ca7c658c2913e4504a0dfd5a5d6072f ? I like pinning because it keeps both lines local rather than needing to split attention?

Reply:
Pinning works for me, definitely makes things simpler.